app/controllers/events/view/edit/attendee.js

```diff
@@ -0,0 +1,31 @@
+import Controller from '@ember/controller';
+import EventWizardMixin from 'open-event-frontend/mixins/event-wizard';
+
+export default Controller.extend(EventWizardMixin, {
+  async saveForms(data) {
+    for (const customForm of data.customForms ? data.customForms.toArray() : []) {
+      try {
```

Comment:
@uds5501 instead of doing it here for each loop, do it at the point where you call `saveForms`. That way even if one fails, others after stop.

Reply:
@niranjan94 Okay!

Reply:
@niranjan94 fixed the implementation, please review